### ISSUE TITLE
15.0.3RC1 Local External Storage Appears Disabled

### ISSUE THREAD

**Issue by jaark** [nextcloud/server#14022]
### Steps to reproduce
1. Create an External Storage mount of type 'Local'
2. Return to the Files app

### Expected behaviour
Mount should be listed in the folder list and be accessible.

### Actual behaviour
The folder is displayed in grey with css class 'externalDisabledRow and does not open if clicked.
The folder is accessible if you manually browse to the URL or use the phone app etc. so I think this is just a web UI layer bug.

### Server configuration

**Operating system**: Ubuntu 16.04

**Web server:** Apache/2.4.18 (Ubuntu) SVN/1.9.3 mod_fcgid/2.3.9 OpenSSL/1.0.2g

**Database:** mysql  Ver 14.14 Distrib 5.7.24

**PHP version:**PHP 7.0.32-0ubuntu0.16.04.1

**Nextcloud version:** 15.0.3RC1

**Updated from an older Nextcloud/ownCloud or fresh install:** Updated

**Where did you install Nextcloud from:** Manual install

**Signing status:**
<details>
<summary>Signing status</summary>

```
No errors have been found.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
Enabled:
  - accessibility: 1.1.0
  - activity: 2.8.2
  - admin_audit: 1.5.0
  - apporder: 0.6.0
  - audioplayer: 2.6.0
  - bookmarks: 0.16.1
  - bruteforcesettings: 1.3.0
  - calendar: 1.6.4
  - cloud_federation_api: 0.1.0
  - comments: 1.5.0
  - contacts: 3.0.3
  - dav: 1.8.1
  - deck: 0.5.2
  - federatedfilesharing: 1.5.0
  - federation: 1.5.0
  - files: 1.10.0
  - files_downloadactivity: 1.4.0
  - files_external: 1.6.0
  - files_fulltextsearch: 1.2.5
  - files_pdfviewer: 1.4.0
  - files_rightclick: 0.11.0
  - files_sharing: 1.7.0
  - files_texteditor: 2.7.0
  - files_trashbin: 1.5.0
  - files_versions: 1.8.0
  - files_videoplayer: 1.4.0
  - firstrunwizard: 2.4.0
  - fulltextsearch: 1.2.4
  - fulltextsearch_elasticsearch: 1.2.3
  - gallery: 18.2.0
  - logreader: 2.0.0
  - lookup_server_connector: 1.3.0
  - mail: 0.11.0
  - mailaccounts: 0.0.8
  - metadata: 0.9.0
  - nextcloud_announcements: 1.4.0
  - notes: 2.5.1
  - notifications: 2.3.0
  - oauth2: 1.3.0
  - provisioning_api: 1.5.0
  - radio: 0.6.4
  - ransomware_protection: 1.3.0
  - serverinfo: 1.5.0
  - sharebymail: 1.5.0
  - spreed: 5.0.2
  - support: 1.0.0
  - survey_client: 1.3.0
  - systemtags: 1.5.0
  - tasks: 0.9.8
  - theming: 1.6.0
  - twofactor_backupcodes: 1.4.1
  - twofactor_totp: 2.1.0
  - twofactor_yubikey: 0.4.1
  - unsplash: 1.1.3
  - updatenotification: 1.5.0
  - user_ldap: 1.5.0
  - workflowengine: 1.5.0
Disabled:
  - bookmarks_fulltextsearch
  - encryption
  - password_policy
  - social
  - user_external
  - weather

```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
{
    "system": {
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "15.0.3.0",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "loglevel": 2,
        "theme": "",
        "maintenance": false,
        "overwrite.cli.url": "\/owncloud",
        "overwritewebroot": "\/owncloud",
        "overwriteprotocol": "https",
        "overwritehost": "home.sexton-family.uk",
        "auth.bruteforce.protection.enabled": true,
        "trusted_proxies": "***REMOVED SENSITIVE VALUE***",
        "blacklisted_files": [
            ".htaccess"
        ],
        "trusted_domains": [
            "wintermute.tr.sexton-family.uk",
            "wintermute.tr.radsnet",
            "home.jack.org.uk",
            "home.sexton-family.uk",
            "home.tr.sexton-family.uk",
            "home.sexton-family.co.uk",
            "home.tr.sexton-family.co.uk"
        ],
        "secret": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "25",
        "appstore.experimental.enabled": true,
        "ldapProviderFactory": "\\OCA\\User_LDAP\\LDAPProviderFactory",
        "share_folder": "\/SharedWithMe\/",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "updater.release.channel": "beta"
    }
}

```
</details>

**Are you using external storage, if yes which one:** local

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** LDAP

### Client configuration
**Browser:** Firefox 65.0 (64-bit), Chromium Version 71.0.3578.98

**Operating system:** Ubuntu 18.04

#### Browser log
<details>
<summary>Browser log</summary>

```
Console Log : 
Content Security Policy: Directive ‘child-src’ has been deprecated. Please use directive ‘worker-src’ to control workers, or directive ‘frame-src’ to control frames respectively.
JQMIGRATE: Migrate is installed, version 1.4.0 core.js:7:542
window.controllers/Controllers is deprecated. Do not use it for UA detection. merged.js:2176
Error: searchBox not available search.js:62:11
querying Whats New data was successful: nocontent merged-template-prepend.js:4613:4
undefined merged-template-prepend.js:4614:4 
```
</details>

**Comment by lefty78** [comment]
I have the same bug

**Comment by MorrisJobke** [comment]
@icewind1991 @nickvergessen @rullzer I noticed that as well :/

**Comment by lefty78** [comment]
The Bug is still in version Nextcloud 15.0.3

**Comment by nickvergessen** [comment]
Yes it will be fixed for 15.0.4

We just noticed this too late. We will post a fix here ASAP. Note that only the webUI is affected. Syncing works just fine.

**Comment by MorrisJobke** [comment]
I found it source - it's caused by https://github.com/nextcloud/server/pull/13891


cc @skjnldsv

**Comment by MorrisJobke** [comment]
Fix for this is in #14076 and we try to release a 15.0.4 today that only contains this fix.

**Comment by lefty78** [comment]
Works,, thx
